views/applications/_form.html

```diff
@@ -0,0 +1,13 @@
+<form action="{{action}}" method="post" enctype="multipart/form-data" class="form" novalidate>
+  <div class="form--field-container">
+    {{form.html | safe}}
+  </div>
+
+  <div class="form--actions">
+    <input type="submit" name="submit" value="{{button}}" class="form--submit">
+    {% if delete %}
+      <input type="submit" name="submit" value="{{config.actions.delete}}" class="form--submit">
+    {% endif %}
+    <a href="../" class="form--cancel">Cancel</a>
+  </div>
+</form>
```

Comment:
Well my point is [that already exist](https://github.com/punchcard-cms/punchcard/blob/master/views/users/_form.html) and is identical, save some spacing and the `delete` if statement. Instead of duplicating work, update that one, and use it. We can spin them out into a shared basic form later